src/renderer/zbay/transit.js

```diff
@@ -478,15 +489,21 @@ export const unpackMemo = async memo => {
         createdAt
       }
     default:
-      const message = memoBuff.slice(timestampEnds)
+      const onionAddress = memoBuff.slice(timestampEnds, timestampEnds + 56)
+      const zcashAddress = memoBuff.slice(timestampEnds + 56, timestampEnds + 56 + 78)
+
+      const message = memoBuff.slice(timestampEnds + 56 + 78, MEMO_SIZE -1)
```

Comment:
Rather use variables that instead of hardcoded values e.g. ONION_ADDRESS_SIZE instead 56